test/nonnegative.jl

```diff
@@ -0,0 +1,64 @@
+using MathOptInterface
+const MOI = MathOptInterface
+const MOIB = MOI.Bridges
+
```

Comment:
It wouldn't hurt to add a comment at the top of the file explaining that this is an example bridge used for tests.